policyengine_us/parameters/gov/states/ar/tax/income/credits/additional/amount.yaml

```diff
@@ -0,0 +1,168 @@
+description: Arkansas provides an additional tax credit of this amount, based on taxable income.
+metadata:
+  threshold_unit: currency-USD
+  type: single_amount
+  amount_unit: currency-USD
+  label: Arkansas additional tax credit amount
+  period: year
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: Arkansas 2023 Individual Income Tax Forms and Instructions 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2023_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: Act 1 of the Second Extraordinary Session
+      href: https://www.arkleg.state.ar.us/Home/FTPDocument?path=%2FACTS%2F2021S2%2FPublic%2FACT1.pdf
+    - title: Act 2 of the Second Extraordinary Session
+      href: https://www.arkleg.state.ar.us/Home/FTPDocument?path=%2FACTS%2F2021S2%2FPublic%2FACT2.pdf
+    - title: Arkansas Codes 26-51-501. Personal tax credits
```

Comment:
im not seeing these parameters here. is this where they'd go if the Act 1 and Act 2 are reflected? if so please add a comment

Reply:
this is the official legal code of AR. the additional credit is mentioned under (a)6(A) section and the amount of the credits can be accessed by clicking the hyperlink